backend/routes/Terraform.ts

```diff
@@ -92,18 +90,20 @@ const cloneGitRepo = async (repositoryPath: string, url: string, authHeader?: st
     }
 };
 
+const removeGitRepo = (repositoryPath: string) => {
+    fs.rmdirSync(repositoryPath, { recursive: true });
+};
+
 const getModuleListForZipBuffer = async (content: Buffer): Promise<string[]> =>
     decompress(content)
         .then((files: File[]) =>
             _(files)
-                .filter({ type: 'directory' })
-                .map('path')
-                .filter(directory =>
-                    files.some(
-                        file => file.type === 'file' && file.path.match(`^${escapeRegExp(directory)}[^/]+\\.tf$`)
-                    )
-                )
-                .map(directory => trimEnd(directory, '/'))
+                .filter(file => file.type === 'file' && file.path.endsWith('.tf'))
+                .map(file => {
+                    const parentPath = path.dirname(file.path);
+                    return parentPath;
+                })
```

Comment:
Optional:

Parent path sounds a bit odd to me. It's directory path extracted from the full path, right?

I would then simplify it to:
```suggestion
                .map(file => path.dirname(file.path)
```

or you can probably make it even shorter by importing `dirname` from `path` and having:

```suggestion
                .map(file => dirname(file.path)
```

Reply:
> It's directory path extracted from the full path, right?
Yeah 😁 

The first approach seems good to me, I'll implement it right away :)